assets/model_monitoring/components/src/feature_importance_metrics/compute_feature_importance.py

```diff
@@ -179,30 +191,31 @@ def compute_feature_importance(task_type, target_column, baseline_data, categori
     model_wrapper = get_model_wrapper(task_type, target_column, baseline_data)
 
     baseline_explanations = compute_explanations(
-        model_wrapper, baseline_data, categorical_features, target_column, task_type)
+        local, model_wrapper, baseline_data, categorical_features, target_column, task_type)
     log_time_and_message("Successfully computed explanations for dataset")
 
     return baseline_explanations
 
 
-def write_to_mltable(explanations, dataset, file_path, categorical_features):
-    """Write feature importance values to mltable.
+def create_signal_metrics_df(global_explanations, dataset, categorical_features):
```

Comment:
nit: Could you rename this to `create_global_feature_importantance_metrics_df` or something similar to better describe what this actually is?

Reply:
From model monitor perspectives, both local and global feature importances are "signal metrics".